python/flexi_cfg_py.cpp

```diff
@@ -179,7 +180,20 @@ PYBIND11_MODULE(flexi_cfg, m) {
                                                                       pyFlexiCfgException);
 
   py::class_<flexi_cfg::Reader>(m, "Reader")
-      .def("dump", &flexi_cfg::Reader::dump)
+      .def(
+          "json",
```

Comment:
this seemed the most "pythonic" way to expose the JSON capability rather than exposing the `visitor` API, ultimately folks want to work with python dicts anyway, which is now possible via: `json.loads(reader.json())`